6911-toolkit-doc (#660)
Esri/arcgis-maps-sdk-toolkit-qt: uitools/toolkitwidgets/docs/AuthenticationView.md

[![ArcGIS Maps SDK for Qt](https://img.shields.io/badge/ArcGIS%20Maps%20SDK%20for%20Qt-0b5394)](https://developers.arcgis.com/qt/) [![ArcGIS Maps SDK for Qt toolkit](https://img.shields.io/badge/ArcGIS%20Maps%20SDK%20for%20Qt%20toolkit-ea4d13)](https://github.com/Esri/arcgis-maps-sdk-toolkit-qt) [![Qt Widgets UI components](https://img.shields.io/badge/Qt%20Qt%20Widgets%20UI%20components-ea4d13)](../../toolkitwidgets/)

> # AuthenticationView - Qt Widgets UI control/Class

Reviewer: Not sure what "control/class" refers to. I think you could probably just leave it at "Qt Widgets UI"

Author: Fixed with commit: https://github.com/Esri/arcgis-maps-sdk-toolkit-qt/pull/660/commits/94d39feb83d333ec756547ac9cf37d3ee93bdec7

> The [AuthenticationView - Qt Widgets UI control/Class](https://developers.arcgis.com/qt/toolkit/api-reference/esri-arcgisruntime-toolkit-authenticationview.html) handles authentication challenges and automatically launching the appropriate UI for each type of authentication. Declare an AuthenticationView in your QML file. The AuthenticationView will then be connected to all authentication challenges, and will automatically launch the appropriate view for the type of challenge. Supported security formats include:

Reviewer: Qt widgets doesn't use QML for the UI, it's all in C++ using "widgets"

```suggestion
The [AuthenticationView - Qt Widgets UI control/Class](https://developers.arcgis.com/qt/toolkit/api-reference/esri-arcgisruntime-toolkit-authenticationview.html) handles authentication challenges and automatically launching the appropriate UI for each type of authentication. Declare an AuthenticationView widget. The AuthenticationView will then be connected to all authentication challenges, and will automatically launch the appropriate view for the type of challenge. Supported security formats include:
```

Author: Thanks for the clarification. Accepted the auto-commit suggestion.
